core/src/main/contraband/ClientAuth.contra

```diff
@@ -0,0 +1,8 @@
+package org.http4s.tls
```

Comment:
I've been threatening to write a developers' guide.  Maybe that would be a good place.